openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -766,7 +766,7 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
 
 The Client MAY request encrypted responses by providing its encryption parameters in the Credential Request.
 
-The Credential Issuer MAY require encrypted responses by including `require_credential_response_encryption` in the Credntial Issuer Metadata.
+The Credential Issuer MAY require encrypted responses by including `credential_response_encryption` object in the Credential Issuer Metadata.
```

Comment:
```suggestion
The Credential Issuer MAY require encrypted responses by including `credential_response_encryption` in the Credential Issuer Metadata.
```

Reply:
If we need to refer to this, we should use the word "parameter" as in Issuer Metadata parameter, Authorization Server metadata parameter, etc. "metadata values" might be also fine.

Reply:
```suggestion
The Credential Issuer MAY require encrypted responses by including `credential_response_encryption` parameter in the Credential Issuer Metadata.
```